diff --git a/node_modules/ethers-multicall/dist/provider.js b/node_modules/ethers-multicall/dist/provider.js
index 56ca1a2..9831cc3 100644
--- a/node_modules/ethers-multicall/dist/provider.js
+++ b/node_modules/ethers-multicall/dist/provider.js
@@ -87,6 +87,7 @@ var multicallAddresses = {
     5: '0x77dca2c955b15e9de4dbbcf1246b4b85b651e50e',
     42: '0x2cc8688c5f75e365aaeeb4ea8d6a480405a48d2a',
     56: '0x1Ee38d535d541c55C9dae27B12edf090C608E6Fb',
+    66: '0x94fEadE0D3D832E4A05d459eBeA9350c6cDd3bCa',
     100: '0xb5b692a88bdfc81ca69dcb1d924f59f0413a602a',
     137: '0xc4f1501f337079077842343Ce02665D8960150B0',
     1337: '0x77dca2c955b15e9de4dbbcf1246b4b85b651e50e',
